android/app/src/main/java/com/google/codeu/chatme/presenter/ProfilePresenter.java

```diff
@@ -82,7 +82,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
      */
     public void getUserProfile() {
         mRootRef.child("users").child(FirebaseUtil.getCurrentUserUid())
-                .addListenerForSingleValueEvent(new ValueEventListener() {
+                .addValueEventListener(new ValueEventListener() {
```

Comment:
Changed this because the Cloud Function takes ~ 200 ms to set the username. Profile's username would not show up right after sign up if we were to stick with `addListenerForSingleValueEvent`.